floss/main.py

```diff
@@ -394,53 +323,22 @@ def is_supported_file_type(sample_file_path):
         return False
 
 
-def print_identification_results(sample_file_path, decoder_results):
-    """
-    Print results of string decoding routine identification phase.
-    :param sample_file_path: input file
-    :param decoder_results: identification_manager
-    """
-    # TODO pass functions instead of identification_manager
-    candidates = decoder_results.get_top_candidate_functions(10)
-    if len(candidates) == 0:
-        print("No candidate functions found.")
-    else:
-        print("Most likely decoding functions in: " + sample_file_path)
-        print(
-            tabulate.tabulate(
-                [(hex(fva), "%.5f" % (score,)) for fva, score in candidates], headers=["address", "score"]
-            )
-        )
-
-
-def print_decoding_results(decoded_strings: List[DecodedString], group_functions, quiet=False, expert=False):
+def print_decoding_results(decoded_strings: List[DecodedString], quiet=False, expert=False):
```

Comment:
yeah `expert` and `quiet` are now basically gone; however, in order to fix them most of the print/render functions change and that would lead into the render refactor. so, i'll remove the cli args, options, and function arguments in that pass.